Github pages (#31)
Supercolony-net/openbrush-contracts: doc/src/examples/README.md

## Overview

> This doc contains example of how the library can be use & how to customize the base implementation.

Reviewer: ```suggestion
This doc contains examples of how the library can be used & how to customize the base implementation.
```

* [PSP22](psp22.md) shows an example of how you can reuse the implementation of

>   [psp22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22) token(by the same way you can reuse

Reviewer: ```suggestion
  [psp22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22) token (in the same way you can reuse
```

>   [psp22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22) token(by the same way you can reuse

Reviewer: invalid link (404)

  [psp721](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp721) and [psp1155](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155)).
* [Access Control](access-control.md) shows how you can use the implementation of
  [access-control](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/access/access-control) and
  [psp721](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp721) together to provide rights to mint and burn NFT tokens.
* [Ownable](ownable.md) shows how you can use the implementation of
  [ownable](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/access/ownable) and
  [psp1155](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155) together to provide rights to mint and burn tokens.
* [ReentrancyGuard](reentrancy-guard.md) shows how you can use the implementation of

>   [non_reentrant](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/security/reentrancy-guard)

Reviewer: invalid link

  modifier to prevent reentrancy during certain functions.
* [Pausable](pausable.md) shows how you can use the implementation of

>   [pausable](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/security/pausable)

Reviewer: invalid link

  contract and modifiers.
* [TimelockController](timelock-controller.md) shows how you can use the implementation of

>   [timelock-controller](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/governance/timelock-controller)

Reviewer: invalid link

>   to execute some transaction with some delay via governance.

Reviewer: ```suggestion
  to execute a transaction with some delay via governance.
```
